apps/paper/pages/introduction/philosophy.mdx

```diff
@@ -8,6 +8,6 @@ Users should not have to deal with the burden of interacting with multiple chain
 
 User should have the freedom to express the atomicity of cross-chain interactions (optimistic, strict, conditional) rather than being enforced/limited by the rules of underlying network.
 
-## Modularity and Permissionless
+## Open Source and Permissionless
```

Comment:
ngl but modluar sounds better lol. up to you

Reply:
true! will fix haha